beacon-chain/node/node.go

```diff
@@ -11,13 +11,13 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/prysmaticlabs/prysm/beacon-chain/blockchain"
 	"github.com/prysmaticlabs/prysm/beacon-chain/database"
-	"github.com/prysmaticlabs/prysm/beacon-chain/network"
 	"github.com/prysmaticlabs/prysm/beacon-chain/powchain"
 	rbcSync "github.com/prysmaticlabs/prysm/beacon-chain/sync"
```

Comment:
s/rbcSync/rbcsync? 

I think the style guide doesn't recommend mixed case for package name
https://blog.golang.org/package-names